refactor/feature(refactored login/signup, made verification, need help with routing ASAP."

emilmoryakov opened the pull request.

rumenmitov committed:
hotfix: dockerfile now builds on the server

rumenmitov committed:
hotfix: dockerfile now builds on the server

sashwatS committed:
refactor (aboutuser, closes #169)  kamran deleted the pr ffs (#197)

* first commit

* save 2

* save 1

* penis

* save 5

* message 1

* checkpoint 69

* checkpoint 70

* checkpoint 71

* final checkpoint hopefully

* checkpoint 73

* Update Sidebar.tsx

* Update AboutUser.tsx

* Update App.tsx

* Update Topbar.tsx

emilmoryakov committed:
First commit: refactor login verification flow

emilmoryakov committed:
Added Refactor + Verification, need help with routing.

emilmoryakov committed:
Added Refactor + Verification, need help with routing.

Nikolay-Ts reviewed (changes requested):
- the text needs to be inverted (light in dark mode and vice versa)
- After the user has clicked log in or register, we need to check if they are verified, if verified => dashboard else => verifications page
- in verifications you should obtain the email via useUser()
- remove app.tsx via git, I will add the verified route as "verification"

Nikolay-Ts committed:
updated port + env

Harishi0375 committed:
removed channels (#202)

removed redundant viewTeam since its called viewProject and that file is there in /Components
updated Dashboard to use viewProject
updated readme to menation where it redirects to

NrdKejsi committed:
fix(repaired login to work properly) (#201)

* s

* only changed login to work, no errors/other files changed

* login.tsx update

* all fixed

* all fixed 2

emilmoryakov committed:
Hopefully works inshAllah

Nikolay-Ts committed:
working docker

Nikolay-Ts committed:
working docker

rumenmitov committed:
hotfix: dockerfile

emilmoryakov committed:
eh

emilmoryakov committed:
Tried to fix

emilmoryakov committed:
merged:

Nikolay-Ts reviewed (changes requested):
- branch is out of date
- best change is to just make a new one IMO

emilmoryakov committed:
Inverted colors

emilmoryakov committed:
updated with webpack package

emilmoryakov committed:
Merged with dev again.

emilmoryakov committed:
Fixed everything, last push i hope

emilmoryakov committed:
Fixed routing

Nikolay-Ts committed:
Merge branch 'dev' into emil-refactor-verify-login

Nikolay-Ts committed:
fixed routing

Nikolay-Ts reviewed (approved):
![](https://duckduckgo.com/?q=approved+knuckles+meme&t=osx&ia=images&iax=images&iai=https%3A%2F%2Fi.imgflip.com%2F7bqii3.jpg)
